viewing_party/party.py

```diff
@@ -0,0 +1,214 @@
+def create_movie(title, genre, rating):
+    if title and genre and rating:
+        # Return a dictionary that has three key-value pairs
+        # The three keys are title, genre, and rating
+        movie_dict = {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+        }
+        return movie_dict
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    watched_list = user_data["watched"]
+    watched_list.append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    add_watchlist = user_data["watchlist"]
+    add_watchlist.append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    movies_watched = user_data["watched"]
+    movies_to_watch = user_data["watchlist"]
+    remove_list = []
+
+    for movie in movies_to_watch:
+        if title == movie["title"]:
+            movies_watched.append(movie)
+            remove_list.append(movie)
+
+    for movie in remove_list:
+        movies_to_watch.remove(movie)
+
+    return user_data
+
+def get_watched_avg_rating(user_data):
+    watched_list = user_data["watched"]
+    total_rating = 0
+    num_ratings = 0
+    avg_rating = 0
+    
+    if watched_list == []:
+        avg_rating = 0.0
+        return avg_rating
+    
+    for movie in watched_list:
+        if movie["rating"]:
+            total_rating += movie["rating"]
+            num_ratings += 1
+        else:
+            total_rating += 0
+            num_ratings += 1
+    
+    avg_rating = total_rating/num_ratings
+    return avg_rating
```

Comment:
Instead of creating a counter variable, you can use `len(user_data["watched"]` to provide the total number of ratings
```suggestion
    watched_list = user_data["watched"]
    total_rating = 0
    avg_rating = 0
    
    if watched_list == []:
        avg_rating = 0.0
        return avg_rating
    
    for movie in watched_list:
        if movie["rating"]:
            total_rating += movie["rating"]
        else:
            total_rating += 0
    
    avg_rating = total_rating/len(user_data["watched"]
    return avg_rating
```